tests/test_dataclasses.py

```diff
@@ -2547,3 +2547,28 @@ class Model(BaseModel, Base):
         pass
 
     assert Model().a == 5
+
+
+def test_alias_with_dashes():
+    """Test for fix issue #7226."""
+    # WITH
```

Comment:
please remove `# WITH` and `# WHEN` and `# THEN`. as you can see in the other tests we are not using them